routers/matchTeamRouter.js

```diff
@@ -13,25 +13,16 @@ exports.getRouter = function () {
   const router = express.Router()
 
   router.get('/:team/matches', (req, res) => {
+    MsLogger.debug(req.params.team)
```

Comment:
This log message need to be more indicative then just a number, it it shouldn't exist

Reply:
done